src/graph/service/PermissionCheck.cpp

```diff
@@ -115,7 +114,8 @@ Status PermissionCheck::permissionCheck(ClientSession *session,
     case Sentence::Kind::kUpdateEdge:
     case Sentence::Kind::kDeleteVertices:
     case Sentence::Kind::kDeleteTags:
-    case Sentence::Kind::kDeleteEdges: {
+    case Sentence::Kind::kDeleteEdges:
+    case Sentence::Kind::kAdminJob: {
```

Comment:
@panda-sheep  Do you think it's okay if I reuse canWriteDate for kAdminJob?

Reply:
reuse  `canWriteDate `for `kAdminJob`  is ok